_posts/2023-07-13-correlating-security-events.md

```diff
@@ -0,0 +1,123 @@
+
+The Security Analytics’ **Correlation Engine** helps you analyze different security event logs and identify relationships between them to find highly correlated events. This blog details how Security Analytics’ can provide a better fidelity of data when viewed through a correlation graph, which can help security analysts monitor their systems and applications more effectively.
+
+
+## What is Security Analytics?
+
+
+With many enterprises bringing an increasing number of critical workloads online, Security Analytics reduces the complexity and cost of running security operations. It achieves this through the Security Analytics’ threat detection engine, which comes pre-loaded with a set of detection rules that you can customize and extend based on your security needs. You can further optimize the threat detection engine to raise only actionable alerts by identifying even sequences that match a standard or custom attack pattern. Detectors are also classified into popular frameworks, such as the MITRE ATT&CK framework, and assigned a risk score which can be further adjusted from external threat intelligence sources. 
+
+When you ingest data from various log sources, Security Analytics dashboards show a real-time view of all threat detections classified by threat type. A user can add log ingestion from one of their existing security products or directly ingest from various supported log sources. The pre-packaged rules are executed against the log data to generate findings / threats, which are represented on a visual dashboard with severity, category and actionable steps/playbooks. You can author new threat rules or use the log pattern detection capabilities to automatically create a threat detection rule.
+
+Before we dive deeper on Security Analytics and correlation engine, it’s useful to know a couple of important terms related to threat detection.
+
+
+### Detectors
+
+**Detectors** are core components that are configured to identify a range of cybersecurity threats corresponding to an ever-growing knowledge base of adversary tactics and techniques maintained by the [MITRE ATT&CK](https://attack.mitre.org/) organization. Detectors use log data to evaluate events occurring in the system. They then apply a set of security rules specified for the detector and determine findings from these events. 
+
+
+### Findings
+
+**Findings** are generated every time a detector matches a rule with a log event. Findings do not necessarily point to imminent threats within the system, but they always isolate an event of interest. Because they represent the result of a specific definition for a detector, findings include a unique combination of select rules, a log type, and a rule severity. As such, you can search for specific findings in the Findings window, and you can filter findings in the list based on severity and log type.
```

Comment:
```suggestion
**Findings** are generated every time a detector matches a rule with a log event. Findings do not necessarily point to imminent threats within the system, but they always isolate an event of interest. Because they represent the result of a specific definition for a detector, findings include a unique combination of selected rules, a log type, and a rule severity. As such, you can search for specific findings in the Findings window, and you can filter findings in the list based on severity and log type.
```